pom.xml

```
@@ -23,10 +24,10 @@
     </licenses>	
 
     <properties>
-        <jenkins.version>1.651.3</jenkins.version>
-        <java.level>7</java.level>
+        <jenkins.version>2.361.4</jenkins.version>
```

This choice of Jenkins version goes against the guidance in ["Choosing a Jenkins version to build against"](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/).  Jenkins 2.479.3 is a better choice because it requires Java 17 and is much closer to recent Jenkins versions.  Jenkins 2.504.3 is also a good choice.
```suggestion
        <jenkins.baseline>2.479</jenkins.baseline>
        <jenkins.version>${jenkins..version}.3</jenkins.version>
```

---

I second that. Consider that 2.361.4 is over 3 years old.